### PR TITLE
Updates to sections

### PR DESCRIPTION
**Pull request by jpradocueva**

### CODE REVIEW

File: content/en/md2html-setup.md [hunk 1]
```diff
@@ -48,3 +48,44 @@ Example of content inside of a `index.md`
     * File name without extension format. See example above
 * `files`
     * This property, list all the names of the markdown documents to display in the order in which should appear on the final document
```
Comment (gocadimic) [inline content/en/md2html-setup.md:50]: Suggestion: This property represents a list of documents in the markdown that needs to be combined to create a final document.

File: content/en/md2html-setup.md [hunk 2]
```diff
@@ -48,3 +48,44 @@ Example of content inside of a `index.md`
...
+
+## MD2HTML v2.x
+
+In v2.0 the file name extension was changed from `.md` to `.yaml`. The `index.yaml` file MUST added to the root folder for each document.
```
Comment (gocadimic) [inline content/en/md2html-setup.md:54]: ...MUST be added.

File: content/en/md2html-setup.md [hunk 3]
```diff
@@ -48,3 +48,44 @@ Example of content inside of a `index.md`
...
+        - Appendix_H.md
+        - Appendix_I.md
+        - Appendix_J.md
+        - Appendix_K.md
+        - Appendix_L.md
+        - Appendix_M.md
+	---
+### New Properties:
+* `organizationName`:  represents the name of the organization.
+* `copyrightDate`:         the copyright date now is defined by this property
```
Comment (gocadimic) [inline content/en/md2html-setup.md:87]: "Now" should be omitted.

File: content/en/md2html-setup.md [hunk 4]
```diff
@@ -48,3 +48,44 @@ Example of content inside of a `index.md`
...
+* `logo`:                          this allows each document to have its own logo
+* `watermark`:               this property inserts a watermark inside of the PDF. Possible values are: "DRAFT",  "CANDIDATE", "HISTORIC", "DEPRECATED", "PRE-APPROVED", "APPROVED".
+
+<alert>In MD2HTML v2.0, the `License.md` contains the license defined for that particular document. It can be inserted at front of the list of files or at the botoom.</alert>
```
Comment (gocadimic) [inline content/en/md2html-setup.md:91]: It should be top and bottom of the list.

File: content/en/paragraph.md [hunk 5]
```diff
@@ -4,12 +4,10 @@ description: ''
 position: 150
 category: MD2HTML App
 ---
-## Paragraph
+This section describes how use `paragraphs` constructors in markdown documents and how those constructors are rendered in `PDF` and `HTML`.
```
Comment (gocadimic) [inline content/en/paragraph.md:7]: It should be "how to use."

File: content/en/table.md [hunk 6]
```diff
@@ -5,29 +5,30 @@ position: 170
...
-Insert basic HTML table
 
-## Tables in HTML
+<alert type="warning">If the table is large with `cells` merged among rows and columns, then it is advisable to create tables using `HTML`.
+Also, if it is important that the tables are incorporated in the Index of the document, then you must construct your tables using `HTML`.</alert>
 
-It is recommended to build `tables` using HTML constructors only. This is because tables will be added automatically to the `Index` section, as long as the `table` is defined with with the `caption` property.
+## HTML Tables
+
+* It is recommended to build `tables` using HTML constructors only. This is because tables will be added automatically to the `Index` section, as long as the `table` is defined with with the `caption` property.
```
Comment (gocadimic) [inline content/en/table.md:31]: One of the "with" should be deleted.

**Comment by jpradocueva** [comment]
@gocadimic, I am ok with the changes but I don't see any PR. If you don't mind could you please create a PR? Thanks

File: content/en/embed_files.md [hunk 7]
```diff
@@ -6,105 +6,130 @@ category: MD2HTML App
...
-* [:SUPP]()
-* [:SUPP1]()
-* [:SUPP3]()
+* [:SUPP](#:supp)
+* [:SUPP1](#:supp1)
+* [:SUPP3](#:supp3)
 
 ### :SUPP
-**File Name: "OMA-SUP-XML_LWM2M_Access_Control-V1_Z.xml"**
+This instruction inserts an `XML` file inside of an `HTML` document as a table:
```
[the remark this answers is not in the document]
Comment (gocadimic) [reply content/en/embed_files.md:16]: Resolved

**Comment by gocadimic** [comment]
@jpradocueva, proposed changes have been added at PR #30.
Once you merge it here, we can go ahead and proceed further. Thanks.

**Pull request merged**